NAMESPACE

```diff
@@ -9,10 +9,12 @@ export(create_glossed_document)
 export(create_presentation)
 export(create_subannotation)
 export(create_viewer)
+export(crocodile)
 export(df_to_tier)
 export(draw_sound)
 export(draw_spectrogram)
 export(eaf_to_df)
+export(eaf_to_df_mod)
```

Comment:
how have you done it? I can't find an `@export` command in your file...

Reply:
I also am not happy with the new function, I would like to have it as the `eaf_to_df()`. And `git` will nicely show the differences.